docs/products/kafka/howto/connect-with-python.rst

```diff
@@ -0,0 +1,154 @@
+
+Variables
+---------
+
+These are the placeholders you will need to replace in the code samples. The values are from the *Connection information* on the service overview page.
+
+If you are using SSL (remember to choose **Client Certificate** if **Authentication Method** is shown):
+
+=============     =============================================================
+Variable          Description
+=============     =============================================================
+``HOST``          Host name for the connection
+-------------     -------------------------------------------------------------
+``SSL_PORT``      Port number to use
+=============     =============================================================
+
+If you are using SASL (**Authentication Method** should be **SASL**):
+
+=================     =============================================================
+Variable              Description
+=================     =============================================================
+``HOST``              Host name for the connection
+-----------------     -------------------------------------------------------------
+``SASL_PORT``         Port number to use
+-----------------     -------------------------------------------------------------
+``SASL_USERNAME``     User to connect with
+-----------------     -------------------------------------------------------------
+``SASL_PASSWORD``     Password for this user
+=================     =============================================================
+
+Connect a producer
+------------------
+
+Using SSL authentication
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+        from kafka import KafkaProducer
+
+        producer = KafkaProducer(
+            bootstrap_servers=f"{HOST}:{SSL_PORT}",
+            security_protocol="SSL",
+            ssl_cafile="ca.pem",
+            ssl_certfile="service.cert",
+            ssl_keyfile="service.key",
+        )
+
+Using SASL authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+         from kafka import KafkaProducer
+
+         # Choose an appropriate SASL mechanism, for instance:
+         SASL_MECHANISM = 'SCRAM-SHA-256'
+
+         producer = KafkaProducer(
+            bootstrap_servers=f"{HOST}:{SASL_PORT}",
+            sasl_mechanism = SASL_MECHANISM,
+            sasl_plain_username = SASL_USERNAME,
+            sasl_plain_password = SASL_PASSWORD,
+            security_protocol="SASL_SSL",
+            ssl_cafile="ca.pem",
+         )
+
+Connect a consumer
+------------------
+
+Using SSL authentication
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+        from kafka import KafkaConsumer
+
+        consumer = KafkaConsumer(
+            "demo-topic",
+            auto_offset_reset="earliest",
+            bootstrap_servers=f"{HOST}:{SSL_PORT}",
+            client_id = CONSUMER_CLIENT_ID,
+            group_id = CONSUMER_GROUP_ID,
+            security_protocol="SSL",
+            ssl_cafile="ca.pem",
```

Comment:
Should we state that this should point to the file? Including the folder if needed?

Reply:
I've added a note to the end of the `Pre-requisites` section - let me know if you think that's enough.